### PR TITLE
add only

### PR DESCRIPTION
**Pull request by bkamins**
Following discussion on Slack I think it is good to add `only` to complement `first` and `last`.

### CODE REVIEW

File: src/DataFrames.jl [hunk 1]
```diff
@@ -80,6 +80,13 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if VERSION >= v"1.4"
+    import Base.only
+else
+    import Compact.only
```
Comment (ararslan) [inline src/DataFrames.jl:86]:
```suggestion
    import Compat.only
```

Comment (bkamins) [reply src/DataFrames.jl:86]: indeed - it is even more compact

**Comment by bkamins** [comment]
only CI and nightly fails now

File: src/DataFrames.jl [hunk 2]
```diff
@@ -80,6 +80,13 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if VERSION >= v"1.4"
```
Comment (ararslan) [inline src/DataFrames.jl:83]: I'd probably do this as
```suggestion
if isdefined(Base, :only)  # Introduced in 1.4.0
```
Probably matters much less nowadays than it used to in the Julia 0.x days. You're welcome to ignore this suggestion if you prefer what's currently there.

Comment (bkamins) [reply src/DataFrames.jl:83]: Actually I was considering this myself, but then I saw what we had for other methods. I am OK to change to what you propose.

**Review by ararslan — APPROVED** [review]
I was wanting this just yesterday! :smile:

**Comment by bkamins** [comment]
@nalimilan - OK to merge?

File: src/DataFrames.jl [hunk 3]
```diff
@@ -80,6 +80,13 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if isdefined(Base, :only)  # Introduced in 1.4.0
```
Comment (nalimilan) [inline src/DataFrames.jl:83]: I don't to prefer explicit VERSION checks, which Femtocleaner can automatically remove once we drop support for 1.4, but as you like.

**Review by nalimilan — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you! I leave it as is. When we will move to a new LTS I will have to review the whole package anyway.

**Pull request merged**